aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -1076,6 +1078,13 @@ _scaled_mm_out_cuda(const Tensor& mat1, const Tensor& mat2,
         out);
     return out;
   }
+#else
+  // For ROCm, match behavior of f8f8bf16_rowwise type checking, for unit test purposes.
+  if (scaling_choice == ScalingType::RowWise) {
+    Tensor b = mat2;
+    TORCH_CHECK(b.dtype() == at::kFloat8_e4m3fn);
```

Comment:
nit: error message?

Reply:
The TORCH_CHECK here has to match the TORCH_CHECK [here](https://github.com/pytorch/pytorch/blob/8c2aa0c5331f221bf554e21b0685ecec09a413bc/aten/src/ATen/native/cuda/RowwiseScaledMM.cu#L521).  Otherwise, the unit test [here ](https://github.com/pytorch/pytorch/blob/8c2aa0c5331f221bf554e21b0685ecec09a413bc/test/test_matmul_cuda.py#L662-L672)has the wrong regex.